### PR TITLE
Added logging to Verify client RPC

### PR DESCRIPTION
**Pull request by ericm**
Added Verify RPC to client and added logging

### CODE REVIEW

File: gnoi/os/client.go [hunk 1]
```diff
@@ -62,10 +62,13 @@ func (c *Client) Activate(ctx context.Context, version string) error {
 
 // Verify invokes the Verify RPC for the OS service.
 func (c *Client) Verify(ctx context.Context) (version, activationFailMsg string, err error) {
+	request := &pb.VerifyRequest{}
+	utils.LogProto(request)
 	var out *pb.VerifyResponse
-	if out, err = c.client.Verify(ctx, &pb.VerifyRequest{}); err != nil {
+	if out, err = c.client.Verify(ctx, request); err != nil {
```
Comment (loriscn12) [inline gnoi/os/client.go:68]: pass &pb.VerifyRequest{} directly, no var assignment needed IMO.

Comment (ericm) [reply gnoi/os/client.go:68]: I'm not sure how I'll pass the request into `LogProto` when not assigning it

Comment (loriscn12) [reply gnoi/os/client.go:68]: would the following not work?

c.client.Verify(ctx, &pb.VerifyRequest{})

Comment (loriscn12) [reply gnoi/os/client.go:68]: sorry i missed the logging part :P

**Pull request merged**